proto/auth.proto

```diff
@@ -69,8 +69,37 @@ message _GenerateApiTokenRequest {
         Expires expires = 2;
     }
 
-    // session token recieved from `momento login` command
-    string session_token = 3;
+    // Aliases for categories of functionality.
+    enum Role {
```

Comment:
I thought we did not want to use the word Role and wanted to use scope?

Reply:
I don't know that the wording really matters tbh

Reply:
It mattered to @kvcache yesterday so want to make sure he is ok here

Reply:
`Action` is another option

Reply:
role meant something different in the context of yesterday's conversation. I think in this narrow position it doesn't mean something terribly different from "role" but maybe future kenny will disagree.

Reply:
I think it'sfine 🤷